go.mod

```diff
@@ -1 +1,3 @@
-module github.com/alash3al/go-pubsub
+module github.com/nitrado/go-pubsub
```

Comment:
@f0086 the module path should be the same as the main github repo

Reply:
thanks, fixed :)